spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -554,7 +551,7 @@ def test_cython_client(ipyconsole, qtbot):
 def test_tab_rename_for_slaves(ipyconsole, qtbot):
     """Test slave clients are renamed correctly."""
     cf = ipyconsole.get_current_client().connection_file
-    ipyconsole.get_widget()._create_client_for_kernel(cf, None, None, None)
+    ipyconsole.create_client_for_kernel(cf)
```

Comment:
Seems like there is an issue with the usage of `get_client_for_file` inside this method?: https://github.com/spyder-ide/spyder/runs/7199975972?check_suite_focus=true#step:13:866